alerts/sender_with_alert.sh

```diff
@@ -68,16 +94,47 @@ do
     2>&1)
 
   echo "$submit"
+  
+  echo "Waiting $VERIFICATION_WAIT_TIME seconds for verification"
+  sleep $VERIFICATION_WAIT_TIME
 
-  explorer_link=$(echo "$submit" | grep alignedlayer.com | awk '{print $4}')
-  sleep 600
+  # Get all the batches merkle roots
+  batch_merkle_roots=$(echo "$submit" | grep "Batch merkle root: " | grep -oE "0x[[:alnum:]]{64}" | uniq)
 
-  ## Get final balance
-  final_balance=$(aligned get-user-balance --rpc_url $RPC_URL --user_addr  $SENDER_ADDRESS --network $NETWORK 2>&1 | awk '{print $7}')
-  echo "Final balance: $final_balance"
+  batch_explorer_urls=()
+
+  # Fetch the logs of both submission and response
+  from_block_number=$((current_block_number - LOGS_BLOCK_RANGE))
+  if [ $from_block_number -lt 0 ]; then
+    from_block_number=0
+  fi
+
+  total_fee_in_wei=0
+  for batch_merkle_root in $batch_merkle_roots 
+  do
+
+    # Construct the batcher explorer url
+    batch_explorer_url="$EXPLORER_URL/batches/$batch_merkle_root"
+    batch_explorer_urls+=($batch_explorer_url)
+
+    log=$(cast logs --rpc-url $RPC_URL --from-block $from_block_number --to-block latest 'NewBatchV3 (bytes32 indexed batchMerkleRoot, address senderAddress, uint32 taskCreatedBlock, string batchDataPointer, uint256 respondToTaskFeeLimit)' $batch_merkle_root)
+    submission_tx_hash=$(echo "$log" | grep -oE "transactionHash: 0x[[:alnum:]]{64}" | awk '{ print $2 }')
+
+    log=$(cast logs --rpc-url $RPC_URL --from-block $from_block_number --to-block latest 'BatchVerified (bytes32 indexed batchMerkleRoot, address senderAddress)' $batch_merkle_root)
+    response_tx_hash=$(echo "$log" | grep -oE "transactionHash: 0x[[:alnum:]]{64}" | awk '{ print $2 }')
+
+    # Calculate fees for transactions
+    submission_fee_in_wei=$(fetch_tx_cost $submission_tx_hash)
+    response_fee_in_wei=$(fetch_tx_cost $response_tx_hash)
```

Comment:
If the verification is not succeded, this call fails with
```
error: the following required arguments were not provided:
  <TX_HASH>

Usage: cast receipt --rpc-url <URL> <TX_HASH> [FIELD]

For more information, try '--help'.
./alerts/sender_with_alert.sh: line 47: * : syntax error: operand expected (error token is "* ")
```

To test it, you can change VERIFICATION_WAIT_TIME=1